crates/kotlin-ffi/src/lib.rs

```diff
@@ -24,35 +25,27 @@ use {
             ui_fields::UiFields,
         },
         config::Config,
-        transaction::{
-            send::safe_test::{
-                Address as FFIAddress, OwnerSignature as YOwnerSignature,
-                PrimitiveSignature,
-            },
-            Transaction as YTransaction,
+        execution::{
+            send::safe_test::{self, Address as FFIAddress, OwnerSignature},
+            Execution,
+        },
+        smart_accounts::{
+            account_address::AccountAddress as FfiAccountAddress,
+            safe::{SignOutputEnum, SignStep3Params},
         },
     },
 };
 
-#[derive(uniffi::Record)]
-pub struct FFIAccountClientConfig {
-    pub owner_address: String,
-    pub chain_id: u64,
-    pub config: Config,
-}
-
-#[derive(uniffi::Record)]
-pub struct FFITransaction {
```

Comment:
Are Transaction and AccountClientConfig directly exposed via yttrium crate since they have been removed from kotlin-ffi?

Reply:
There is no longer an `AccountClientConfig` as you just pass these parameters directly to [`FFIAccountClient::new(...)`](https://github.com/reown-com/yttrium/pull/101/files#diff-cc41b25f265e3973cc6715f6f8d824a28a0970f381601884edc600305f492a2aR214)